system-tests/lib/cre/don/jobs/standardcapability/donlevel/strategies.go

```diff
@@ -12,9 +12,9 @@ func JobNamer(chainID uint64, flag cre.CapabilityFlag) string {
 	return flag
 }
 
-func CapabilityEnabler(donMetadata *cre.DonMetadata, nodeSet *cre.CapabilitiesAwareNodeSet, flag cre.CapabilityFlag) bool {
+func CapabilityEnabler(capabilities []string, _ *cre.CapabilitiesAwareNodeSet, flag cre.CapabilityFlag) bool {
```

Comment:
🤔 What are these strategies? Mind explaining -- I know it's not related to the current PR, but I don't immediately understand what this is for or does

Reply:
we have a capability factory that has different strategies for different capability "types"
- don-level (e.g. `cron` or `http-actions` - you only ever have 1 per DON)
- chain-level (e.g. `evm` - scoped to chain, you can have multiple per DON)

So that factory we pass different implementations of these functions based on the capability type.